changelogs/unreleased/1672-prateekpandey14

```diff
@@ -0,0 +1 @@
+refact(webhook): update webhookconfiguration failure policy to Fail
```

Comment:
folder name is 1.10.0 ?

Reply:
The unreleased folder has no subfolders. Should we move them to `released/v1.10.0`?

Reply:
the folders should be changelogs/released/v1.10.0-RC1, changelogs/released/v1.10.0-RC2 and changelogs/released/v1.10.0

Reply:
Did the same.